python-sdk/nuscenes/map_expansion/tests/test_all_maps.py

```diff
@@ -68,7 +68,7 @@ def test_egoposes_on_map(self):
         whitelist = ['scene-0499', 'scene-0501', 'scene-0502', 'scene-0515', 'scene-0517']
 
         invalid_scenes = []
-        for scene in tqdm.tqdm(nusc.scene):
+        for scene in tqdm.tqdm(nusc.scene, leave=False):
```

Comment:
Out of curiosity, why add `leave=False` now?

Reply:
If we don't leave it will leave behind the progress bar in the test suite.